sdk/core/azure-core-tracing-opentelemetry/README.md

```diff
@@ -22,11 +22,9 @@ with azure-core tracing. This includes (not exhaustive list), azure-storage-blob
 
   ``` python
     from azure.core.settings import settings
-    from azure.core.tracing.ext.opentelemetry_span import OpenTelemetrySpan
-    settings.tracing_implementation = OpenTelemetrySpan
+    settings.tracing_implementation = "opentelemetry"
```

Comment:
is the other way still valid ( other than mypy or pylint errors ) ? I'm wondering if a # ignore statement would be valid here to show 2 ways

Reply:
Right, it's still valid, and I think it's sensible to show both ways in the README. Will update. 👍

Reply:
sounds good :)